streams/src/test/java/org/apache/kafka/streams/state/internals/NamedCacheTest.java

```diff
@@ -45,7 +44,7 @@
 
     @Before
     public void setUp() throws Exception {
-        streamMetrics = new MockStreamsMetrics(new Metrics());
+        streamMetrics = new MockStreamsMetrics();
```

Comment:
Similarly for `ProcessorNodeTest`: the only difference is that for that test case we can introduce another overloaded constructor for `MockProcessorContext` which takes a `StreamsMetrics` object, and then pass a real `StreamsMetricsImpl` only for that `testMetrics` class while for all others just use the other constructor which always use a `MockStreamsMetrics` object.